applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_small_strain_umat_law.cpp

```diff
@@ -0,0 +1,31 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+#include "custom_constitutive/small_strain_umat_2D_plane_strain_law.hpp"
+#include "custom_constitutive/three_dimensional.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(SmallStrainUMATLaw_HasReturnsTrueForCauchyStress, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    auto umat_law = SmallStrainUMAT2DPlaneStrainLaw{std::make_unique<ThreeDimensional>()};
```

Comment:
Doesn't matter for the test, but since the law is 2d plane strain, I'd use the PlaneStrain ConstitutiveLawDimension to avoid confusion
```suggestion
    auto umat_law = SmallStrainUMAT2DPlaneStrainLaw{std::make_unique<PlaneStrain>()};
```

Reply:
After looking together: No this needs to be ThreeDimensional.